### PR TITLE
Bug 1209041 - `dir` attribute support

### PR DESCRIPTION
**Pull request by fabi1cazenave**
Counterpart of https://github.com/gaia-components/gaia-component/pull/16. I guess we’ll need to adjust the gaia-component version number when the related patch lands.

### CODE REVIEW

File: gaia-header.js [hunk 1]
```diff
@@ -404,7 +404,6 @@ module.exports = component.register('gaia-header', {
    */
   onUpdate: function() {
     debug('onUpdate');
-    this.shadowRoot.firstChild.dir = dir();
```
Comment (fabi1cazenave) [inline gaia-header.js]: @wilsonpage now that `onUpdate` is useless, do you want me to remove it completely (along with the `dirchanged` listeners) or should we leave that here to ease future work?

**Comment by fabi1cazenave** [comment]
As you can see, if we land a simple workaround for bug 1100912 in gaia-component, the RTL support is a snap — we just have to enable `dirObserver` from this component.

File: gaia-header.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ const MAXIMUM_FONT_SIZE = 23;
  * @return {Element} constructor
  */
 module.exports = component.register('gaia-header', {
-  rtl: true,
+  rtl: true, // triggers a workaround for bug 1100912 in gaia-component
```
Comment (wilsonpage) [inline gaia-header.js]: Should be `dirChanged` ?

Comment (fabi1cazenave) [inline gaia-header.js]: it is — I just reverted it to `rtl` in the second commit to see if that was the reason why the tests failed. I’ll remove this second commit when we land the patch on gaia-component.

**Comment by fabi1cazenave** [comment]
@wilsonpage ready to merge?

**Pull request merged**